docs/index.html

```diff
@@ -79,6 +79,8 @@
         <script>
             const RusfmtTagsUrl = 'https://api.github.com/repos/rust-lang/rustfmt/tags';
             const UrlHash = window.location.hash.replace(/^#/, '');
+            const queryParams = new URLSearchParams(window.location.search);
+            const versionNumber = null !== queryParams.get('version') ? 'v' + queryParams.get('version') : 'master';
```

Comment:
This is looking good! I think we need to tweak it a bit though so we don't end up unnecessarily adding the `v` prefix, for example `vmaster` with `?version=master` or `vv1.4.17` with `?version=v1.4.17`. 

What about something like this (obligatory disclaimer about potential issues with GH snippets written inline ):
```suggestion
            const versionParam = queryParams.get('version');
            if (versionParam === null || versionParam === 'master') {
              versionNumber = 'master'
            } else {
              versionNumber = versionParam.startsWith("v") ? versionParam : `v${versionParam}`
            } 
```

Reply:
Thanks for the feedback, I extracted this logic into a function.